FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift

```diff
@@ -43,4 +32,43 @@ class FirefoxPrivateNetworkVPNUITests: XCTestCase {
             }
         }
     }
+
+    func testOnboarding() {
+        // Wait for main page to load where Mozilla VPN label is shown
+        waitForExistence(app.buttons["Learn more"], timeout: 5)
+        XCTAssertTrue(app.staticTexts["Mozilla VPN"].exists)
+
+        // Start the onboaring cards
+        app.buttons["Learn more"].tap()
+
+        // Wait for the first onboarding card shown
+        // Verify the close, skip and page indicator buttons
+        waitForExistence(app.staticTexts["Device-level encryption"], timeout: 3)
+        XCTAssertTrue(app.buttons["icon close"].exists)
+        XCTAssertTrue(app.buttons["Skip"].exists)
+        // XCTAssertTrue(app.pageIndicators["page 1 of 4"].exists)
```

Comment:
This assertion should be removed if it's not been used anymore. Same as line 55 and 60.